pkg/k8sapi/k8sutils.go

```diff
@@ -47,6 +49,7 @@ func CreateKubeClient(mapper meta.RESTMapper) (client.Client, error) {
 	vpcCniScheme := runtime.NewScheme()
 	clientgoscheme.AddToScheme(vpcCniScheme)
 	eniconfigscheme.AddToScheme(vpcCniScheme)
+	rcscheme.AddToScheme(vpcCniScheme)
```

Comment:
Have we measured the number of API server calls increase with this change?

Reply:
We are changing to use rawclient.